### ISSUE TITLE
Unable to install in new framework at Magento 2.1 -> no matching package found.

### ISSUE THREAD

**Issue by christian-eland** [flowecommerce/magento2-pt_br#2]
Your requirements could not be resolved to an installable set of packages.
- flowecommerce/pt_br 100.0.1 requires magento/framework 100.0.* -> no matching package found.
Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://groups.google.com/d/topic/composer-dev/_g3ASeIFlrc/discussion> for more details.

**Comment by caioiglesias** [comment]
same as #1

**Comment by gabrielqs** [comment]
Should be fixed after PR #4 was merged. Please reopen if your problem persists.

**Comment by jrvidotti** [comment]
Same as me. Fresh install of Bitnami Magento 2.1.7 (docker)...

```
Using version ^100.0 for flowecommerce/pt_br
./composer.json has been created
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for flowecommerce/pt_br ^100.0 -> satisfiable by flowecommerce/pt_br[100.0.1].
    - flowecommerce/pt_br 100.0.1 requires magento/framework 100.0.* -> no matching package found.

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://getcomposer.org/doc/04-schema.md#minimum-stability> for more details.

Read <https://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.

Installation failed, deleting ./composer.json.
```
